osu.Game/Screens/SelectV2/WedgeBackground.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Extensions.Color4Extensions;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Colour;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Game.Overlays;
+
+namespace osu.Game.Screens.SelectV2
+{
+    internal partial class WedgeBackground : CompositeDrawable
```

Comment:
added here but not really tested anywhere in this pr. i'll overlook this though as i already know what this is and what it does.